recovery-grub.cfg

```diff
@@ -0,0 +1,28 @@
+set default=0
+set timeout=3
+
+insmod part_gpt
+insmod ext2
+insmod regexp
+
+if [ -s $prefix/grubenv ]; then
+  load_env
+fi
+
+menuentry "Normal mode" {
+          search --label "system-boot" --set=root
+          chainloader +1
+}
+
+set cmdline="root=LABEL=$label snap_recovery_mode=$snap_recovery_mode snap_core=core.snap snap_kernel=kernel.snap ro net.ifnames=0 console=ttyS0 console=tty1 panic=-1"
+
+for system in /system/*
+do
+  menuentry "Recovery $system" $system {
```

Comment:
why the multiple recovery entries, wasn't the plan do the choosing from initramfs?

Reply:
Yes, I changed it to a single static entry.